src/Chainweb/Rosetta/RestAPI/Server.hs

```diff
@@ -119,3 +123,37 @@ mempoolTransactionH mtr ms = runExceptT work >>= either throwRosetta pure
 
 --------------------------------------------------------------------------------
 -- Network Handlers
+
+networkListH :: MetadataReq -> ChainwebVersion -> Handler NetworkListResp
+networkListH _ v
+  | (isRosettaEnabled v) = return $ NetworkListResp (networkIds v)
+  | otherwise = throwRosetta $ RosettaNotSupported v
```

Comment:
Decided to throw an error here, but wanted to get opinion on returning an empty list instead?

Reply:
`isRosettaEnabled` is probably unnecessary in general, since the node can decide if they turn these endpoints on or off. They're off by default.

Reply:
uhm I see what you mean. If Rosetta is disabled in the config, will trying to access `/rosetta/<endpoint>` return a 404 error?

Reply:
Yes.